src/EdFi.AnalyticsMiddleTier.DataStandard32/Base/PostgreSQL/0036-View-StudentLocalEducationAgencyDim-Alter.sql

```diff
@@ -0,0 +1,81 @@
+﻿-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+
+CREATE OR REPLACE VIEW analytics.StudentLocalEducationAgencyDim
```

Comment:
If you look at script 0035 you'll see that there's a DROP statement at the top. I thought that was awfully strange, coming before `CREATE OR REPLACE`. Turns out that PostgreSQL does not want you to alter a view and add columns unless appending to the existing column list. So the recommended practice is to drop the view first.